lib/android-tools-info.ts

```diff
@@ -186,6 +139,51 @@ export class AndroidToolsInfo implements IAndroidToolsInfo {
 		return null;
 	}
 
+	@cache()
+	public validateAndroidHomeEnvVariable(options?: { showWarningsAsErrors: boolean }): boolean {
+		if (options) {
+			this.showWarningsAsErrors = options.showWarningsAsErrors;
+		}
+
+		const expectedDirectoriesInAndroidHome = ["build-tools", "tools", "platform-tools", "extras"];
+		let androidHomeValidationResult = true;
+
+		if (!this.androidHome || !this.$fs.exists(this.androidHome)) {
+			this.printMessage("The ANDROID_HOME environment variable is not set or it points to a non-existent directory. You will not be able to perform any build-related operations for Android.",
+				"To be able to perform Android build-related operations, set the `ANDROID_HOME` variable to point to the root of your Android SDK installation directory.");
+			androidHomeValidationResult = false;
+		} else if (!_.some(expectedDirectoriesInAndroidHome.map(dir => this.$fs.exists(path.join(this.androidHome, dir))))) {
+			this.printMessage("The ANDROID_HOME environment variable points to incorrect directory. You will not be able to perform any build-related operations for Android.",
+				"To be able to perform Android build-related operations, set the `ANDROID_HOME` variable to point to the root of your Android SDK installation directory, " +
+				"where you will find `tools` and `platform-tools` directories.");
+			androidHomeValidationResult = false;
+		}
+
+		return androidHomeValidationResult;
+	}
+
+	@cache()
+	private getPathToSdkManagementTool(): string {
+		const sdkmanagerName = "sdkmanager";
+		let sdkManagementToolPath = sdkmanagerName;
+
+		const isAndroidHomeValid = this.validateAndroidHomeEnvVariable();
+
+		if (isAndroidHomeValid) {
```

Comment:
This method is used just to get the path that we'll show in the `additionalMessage`. The idea is that if something is not configured properly, `tns doctor` command to show some suggestions how to fix the issue. 
In case you execute any other command, CLI will fail immediately in case the ANDROID_HOME is not set. The default is `sdkmanager` as when the ANDROID_HOME is not set, most probably the user does not have any Android SDK. After installing it, he'll have the new SDK, which requires usage of `sdkmanager` not `android`.